layers/+lang/go/packages.el

```diff
@@ -14,6 +14,13 @@
   (when (memq window-system '(mac ns x))
     (exec-path-from-shell-copy-env "GOPATH"))
 
+  (add-hook 'go-mode-hook
+            (lambda ()
```

Comment:
Can you define a function instead of using a lambda ? Example of name could be `spacemacs//set-go-indentation`.